micro-rdk/src/common/app_client.rs

```diff
@@ -43,6 +43,8 @@ use super::{
 #[cfg(feature = "data")]
 use crate::proto::app::data_sync::v1::DataCaptureUploadRequest;
 
+pub const VIAM_FOUNDING_YEAR: i32 = 2020;
```

Comment:
Could this be a const `DateTime` for the beginning of year 2020?

Reply:
resolved offline, would have to use lazy static or some lock mechanism because Rust doesn't have const expr support